new file with useless facts about miencraft

Fusiongames1337 opened the pull request:
some facts about minecraft in a second file  [insert amazed.mp3]

Fusiongames1337 committed:
new file with useless facts about miencraft

some facts about minecraft in a second file  [insert amazed.mp3]

hypergonial reviewed (changes requested):
Punctuation is all over the place, capitalization is also needed.

Fusiongames1337 committed:
fixed spelling and grammar

imagine using the right way to spell things. PATHETIC

might be correct this time

hypergonial committed:
Fix naming

hypergonial committed:
Add the command

HyperGH commented:
Why am I doing this...
